config/config.json

```diff
@@ -909,7 +919,8 @@
         "titleUrlSafeText": "Safe site according to AI (probability <PERCENTAGE>%)",
         "reasonText": "Reason: <REASON>",
         "buttonBadDecisionText": "Bad decision",
-        "checkboxDonotShowText": "Don't show this anymore"
+        "checkboxDonotShowText": "Don't show this anymore",
+        "progressInfoText": "Here is the progress while you are teaching Focus Bear to keep allowing sites/apps but within a few days, you'll have taught Focus Bear what apps and websites are productive."
```

Comment:
You're teaching Focus Bear what sites and apps are productive for you. It takes a bit of effort at first but within a few days, you'll have taught Focus Bear what apps and websites are productive.